api/sanitize_ascii.go

```diff
@@ -90,18 +87,14 @@ func (s *sanitizeASCIIReadCloser) Read(out []byte) (int, error) {
 			s.addBackslash = false
 		}
 
-		out[outIndex] = buf[bufIndex]
+		out = append(out[:outIndex], buf[bufIndex])
```

Comment:
Thank you for quick fix!

This `Read` implementation may return `n` which is bigger than length of provided buffer.

Test case:
```go
[]byte("{\"__typename\":\"\",\"ID\":\"\",\"Number\":0,\"Title\":\"\\u001b[31mRed Title\\u001b[0m\",\"URL\":\"\",\"State\":\"\",\"StateReason\":\"\",\"Closed\":false,\"Body\"\\u0001 2\\u0002 3\\u0003 4\\u0004 5\\u0005 6\\u0006\x0e7\\u0007 8\\u0008 9\\t A\\r\\n B\\u000b C\\u000c D\\r\\n E\\u000e F\\u000f\",\"ActiveLockReason\":\"Escaped \\u001b \\\\u001B \\\\\\u001b \\\\\\\\u001B\",\"Locked\":false,\"CreatedAt\":\"0001-01-01T00:00:00Z\",\"UpdatedAt\":\"0001-01-01T00:00:00Z\",\"ClosedAt\":null,\"Comments\":{\"Nodes\":null,\"TotalCount\":0,\"PageInfo\":{\"HasNextPage\":false,\"EndCursor\":\"\"}},\"Author\":{\"id\":\"1\",\"is_bot\":false,\"login\":\"monalisa\",\"name\":\"10\\u0010 11\\u0011 12\\u0012 13\\u0013 14\\u0014 15\\u0015 16\\u0016 17\\u0017 18\\u0018 19\\u0019 1A\\u001a 1B\\u001b 1C\\u001c 1D\\u001d 1E\\u001e 1F\\u001f\"},\"Assignees\":{\"Nodes\":null,\"TotalCount\":0},\"Labels\":{\"Nodes\":null,\"TotalCount\":0},\"ProjectCards\":{\"Nodes\":null,\"TotalCount\":0},\"ProjectItems\":{\"Nodes\":null},\"Milestone\":null,\"ReactionGroups\":[],\"IsPinned\":false}")
```